scipy/special/generate_ufuncs.py

```diff
@@ -393,13 +393,289 @@ def errprint(inflag=None):
         return sf_error.get_print()
 """
 
-EXTRA_CODE_BOTTOM = """\
+UFUNCS_EXTRA_CODE_BOTTOM = """\
 #
 # Aliases
 #
 jn = jv
 """
 
+CYTHON_SPECIAL_PXD = """\
+# This file is automatically generated by generate_ufuncs.py.
+# Do not edit manually!
+"""
+
+CYTHON_SPECIAL_PYX = '''\
+# This file is automatically generated by generate_ufuncs.py.
+# Do not edit manually!
+"""
+================================
+Cython API for Special Functions
+================================
+
+Scalar, typed versions of many of the functions in ``scipy.special``
```

Comment:
It could be simpler to just list all the available C signatures, rather than the internal representation used in generate.py --- this way it would be more self-explanatory.